backend/attendance/CHANGELOG.md

```
@@ -1,6 +1,22 @@
+
+## 1.1.1 - 2025-06-13
+### Changes
+
+- Added new tag field in eg_wms_attendance_attendee table with supporting index 
+- Introduced support for attendee tagging and tag-based filtering
+- Enhanced attendee retrieval logic to support:
+    - Filtering attendees based on explicit tags
+    - Resolving and including attendees with matching tags when includeTaggedAttendees is true
+- Added new /_updateTag API to update attendee tags
+
+### Configuration Enhancements
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)